src/main/scala/com/databricks/labs/overwatch/utils/Config.scala

```diff
@@ -355,7 +355,7 @@ class Config() {
       if (!rawToken.matches("^(dapi|dkea)[a-zA-Z0-9-]*$")) throw new BadConfigException(s"contents of secret " +
         s"at scope:key $scope:$key is not in a valid format. Please validate the contents of your secret. It must be " +
         s"a user access token. It should start with 'dapi' ")
-      setApiEnv(ApiEnv(isLocalTesting, workspaceURL, rawToken, packageVersion))
+      setApiEnv(ApiEnv(isLocalTesting, workspaceURL, rawToken, packageVersion,1,1,runID))
```

Comment:
why are we force setting batch sizes to 1 here and then later setting it to 50 inside APIMeta. These should be externalized into the Overwatch global config and defaulted to something that makes sense, like 50.

Reply:
done